.github/workflows/sync-api-vmlinux-cron.yaml

```diff
@@ -0,0 +1,99 @@
+name: Sync api.h and vmlinux.h from latest kernel
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 2 1 * *' # first day of each month
+
+jobs:
+  sync-api-vmlinux:
+    runs-on: ${{ (matrix.arch == 'arm64' && 'ubuntu-24.04-arm') || 'ubuntu-24.04' }}
+    strategy:
+      matrix:
+        arch: [ x86, arm64 ]
+        include:
+          - arch: x86
+            tumbleweed_repo: 'https://download.opensuse.org/tumbleweed/repo/oss/x86_64/'
+          - arch: arm64
+            tumbleweed_repo: 'https://download.opensuse.org/ports/aarch64/tumbleweed/repo/oss/aarch64/'
+    steps:
+      - name: Install bpftool
+        uses: mtardy/setup-bpftool@9bc044304197616a6321d452e04cc059e006d8e2 # v1.0.4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Download most recent kernel package from opensuse tumbleweed
+        run: |
+          image=$(wget -qO- ${{ matrix.tumbleweed_repo }} | grep "kernel-default-base-*" | head -n1 | awk '{split($0,file,"\""); print file[4]}')
+          wget -O image.rpm ${{ matrix.tumbleweed_repo }}$image
+          mkdir image && cd image
+          rpm2cpio ../image.rpm | cpio -idmv
+          prefix="./kernel-default-base-"
+          suffix=".rpm"
+          version=${image#"$prefix"}
+          version=${version%"$suffix"}
+          echo "KERNEL_VERSION=$version" >> $GITHUB_ENV
```

Comment:
We also save the downloaded kernel version in github env to later use it as comment to the generated headers.